### PR TITLE
#963 - incomplete documentation

### PR DESCRIPTION
**Pull request by maurezen**
Different tests require different scopes. Added link to github scopes doc and listed scope requirement for the example test.

### CODE REVIEW

**Comment by dmarkov** [comment]
Thanks for your pull request, let me find someone who can review it

**Comment by dmarkov** [comment]
@carlosmiranda please review this PR

**Comment by carlosmiranda** [comment]
@maurezen The explanation is not very clear. Perhaps you should be more explicit. Indicate that `failsafe.github.key` should have permissions to all needed scopes in `failsafe.github.repo` if the entire integration test suite is to be run.

**Comment by maurezen** [comment]
@carlosmiranda agreed, elaborated on that.

File: README.md [hunk 1]
```diff
@@ -74,6 +74,11 @@ $ mvn clean install -Dit.test=RtGistITCase -Dfailsafe.github.key=<token> -Dfails
 
 Replace `<token>` with the OAuth access token, and `<repo>` with the name of
 repository you create in your account (for test purposes only), for example `yegor256/test`.
+The failsafe.github.key should have permissions in failsafe.github.repo to all scopes needed
```
Comment (carlosmiranda) [inline README.md]: Just one more thing - let's mark out `failsafe.github.key` and `failsafe.github.repo` as code, to be consistent with the rest of the document. You can do that by using the ` character in the Markdown code:

```markdown
The `failsafe.github.key` should have permissions in `failsafe.github.repo` to all scopes needed
```

**Comment by carlosmiranda** [comment]
@maurezen Thanks for the update, just one more minor comment.

**Comment by maurezen** [comment]
@carlosmiranda agreed, fixed.

**Comment by carlosmiranda** [comment]
@maurezen Thanks for your update!

**Comment by carlosmiranda** [comment]
@rultor Good to merge

**Comment by yegor256** [comment]
@rultor merge pls

**Comment by dmarkov** [comment]
@rultor please deploy

**Comment by dmarkov** [comment]
@carlosmiranda thanks, I just added **16 mins** to your account, payment `50342787`; +16 to your rating, your total score is [+1288](http://www.netbout.com/b/35023?open=rating)

**Pull request merged**